app/services/fastly_vcl/whitelisted_params.rb

```diff
@@ -0,0 +1,58 @@
+module FastlyVCL
+  # Handles updates to our VCL snippet on Fastly that whitelists params
+  class WhitelistedParams
+    VCL_REGEX_START = "^(".freeze
+    VCL_REGEX_END = ")$".freeze
```

Comment:
While it's highly unlikely there will be a major VCL syntax update causing these delimiters to change, I put these in constants for a few reasons:
1. If the VCL we're trying to update changes or if we add a new VCL script, we can adjust these easily and it will flow through everwhere else.
2. This is very useful to have access to in the test suite vs hardcoding the same thing in two different places.
3. It's a little more clear as to what's going on.